### PR TITLE
Add notifications block to AllAuth templates

### PR DESCRIPTION
**Pull request by ericholscher**
The allauth templates are overwriting the base templates block:

* https://github.com/readthedocs/ext-theme/blob/main/readthedocsext/theme/templates/allauth/layouts/base.html#L3 is overriding
* https://github.com/readthedocs/ext-theme/blob/main/readthedocsext/theme/templates/base.html#L63

Fixes https://github.com/readthedocs/ext-theme/issues/425

### CODE REVIEW

File: readthedocsext/theme/templates/allauth/layouts/base.html [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 {% block content-wrapper %}
   <div class="ui very padded container">
```
Comment (agjohnson) [inline readthedocsext/theme/templates/allauth/layouts/base.html:4]: Not entirely sure, but the HTML structure is the same up until this block. We might be able to just move the override to `content` block instead. This requires removing a closing div below too

```suggestion
{% block content %}
```

**Review by agjohnson — APPROVED** [review]
What you have here definitely should work, though I suspect `content-wrapper` is maybe just the wrong element to override. If we can override `content` instead, that would allow the base notifications/messages to show still.

File: readthedocsext/theme/templates/allauth/layouts/base.html (hunk 1 above)
Comment (ericholscher) [reply readthedocsext/theme/templates/allauth/layouts/base.html:4]: @agjohnson This template is also defining the content inside the wrapper a few lines down, so I think we still need the wrapper here.

File: readthedocsext/theme/templates/allauth/layouts/base.html [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
+
+    {# This is overriding the normal base.html content-wrapper, #}
+    {# so we need to include notifications here so they're displayed. #}
+    {% block notifications %}
+      {% include "includes/utils/messages.html" %}
```
Comment (agjohnson) [inline readthedocsext/theme/templates/allauth/layouts/base.html:9]: This at least avoids duplicating logic

```suggestion
      {{ block.super }}
```

**Pull request merged**